stdlib/LinearAlgebra/src/deprecated.jl

```diff
@@ -1260,3 +1260,10 @@ end
 @deprecate scale!(C::AbstractMatrix, a::AbstractVector, B::AbstractMatrix) mul!(C, Diagonal(a), B)
 
 Base.@deprecate_binding trace tr
+
+# deprecate lufact to lu
+@deprecate(lufact(S::LU), lu(S))
```

Comment:
Since it's a simple renaming rather than an API change, you can do the same thing as was done with `trace` above and just call `Base.@deprecate_binding lufact lu`

Reply:
I hesitate to broaden the deprecated signatures as `*fact` methods quite possibly exist outside stdlib. Thoughts?

Reply:
Deprecations from `@deprecate_binding` are impossible to find since they don't show a location.